### PR TITLE
Testing new diff colors

### PR DESCRIPTION
**Pull request by edokoa**
This is a test to see if I can check the new proposed diff colors in dark mode live. Issue https://github.com/github/design-systems/issues/1168

### CODE REVIEW

**Review by simurai — APPROVED** [review]
Ok, we tested this quick in the browser and contrast should be improved. 👍  🚢 

@edokoa is still gonna review it in the color sync.

File: data/colors/mixins/dark_mode.scss [hunk 1]
```diff
@@ -575,17 +575,17 @@ $export: (
...
+      word-bg: rgba($green-4, 0.55),
     ),
 
     deletion: (
       num-text: $red-4,
       num-hover-text: $red-2,
-      num-bg: rgba($red-5, 0.05),
-      line-bg: rgba($red-5, 0.15),
-      word-bg: rgba($red-5, 0.3),
+      num-bg: rgba($red-5, 0.14),
```
Comment (edokoa) [inline data/colors/mixins/dark_mode.scss:586]: test

```suggestion
      ndfghdfghdfghdfghdfghdfghdfgh
```

File: data/colors/mixins/dark_mode.scss [hunk 2]
```diff
@@ -575,17 +575,17 @@ $export: (
     addition: (
       num-text: $green-3,
       num-hover-text: $green-1,
-      num-bg: rgba($green-5, 0.05),
-      line-bg: rgba($green-5, 0.15),
-      word-bg: rgba($green-5, 0.3),
+      num-bg: rgba($green-4, 0.14),
+      line-bg: rgba($green-4, 0.25),
+      word-bg: rgba($green-4, 0.55),
```
Comment (edokoa) [inline data/colors/mixins/dark_mode.scss:580]:
```suggestion
      num-bg: rgba($green-4, 0.10),
      line-bg: rgba($green-4, 0.20),
      word-bg: rgba($green-4, 0.55),
```

File: data/colors/mixins/dark_mode.scss [hunk 3]
```diff
@@ -575,17 +575,17 @@ $export: (
...
 
     deletion: (
       num-text: $red-4,
       num-hover-text: $red-2,
-      num-bg: rgba($red-5, 0.05),
-      line-bg: rgba($red-5, 0.15),
-      word-bg: rgba($red-5, 0.3),
+      num-bg: rgba($red-5, 0.14),
+      line-bg: rgba($red-5, 0.25),
+      word-bg: rgba($red-5, 0.55),
```
Comment (edokoa) [inline data/colors/mixins/dark_mode.scss:588]:
```suggestion
      num-bg: rgba($red-5, 0.10),
      line-bg: rgba($red-5, 0.20),
      word-bg: rgba($red-5, 0.50),
```

File: data/colors/mixins/dark_mode.scss [hunk 4]
```diff
@@ -575,17 +575,17 @@ $export: (
     addition: (
       num-text: $green-3,
       num-hover-text: $green-1,
-      num-bg: rgba($green-5, 0.05),
-      line-bg: rgba($green-5, 0.15),
-      word-bg: rgba($green-5, 0.3),
+      num-bg: rgba($green-4, 0.10),
+      line-bg: rgba($green-4, 0.20),
+      word-bg: rgba($green-4, 0.55),
```
Comment (edokoa) [inline data/colors/mixins/dark_mode.scss:580]:
```suggestion
      word-bg: rgba($green-4, 0.55),
      test
      test
      test
```

**Comment by edokoa** [comment]
Updated this PR with the current values for diff view fixes as per https://github.com/github/design-systems/discussions/1180

We still have to fix a bug causing diff view comments to have alternating grey line backgrounds

`.blob-wrapper table tr:nth-child(2n) {
  background-color: transparent;
}`

image

@simurai do you know where to fix this or do we leave it for another PR?

**Pull request merged**